libs/application/templates/new-primary-school/src/utils/overviewItems.ts

```diff
@@ -751,3 +752,42 @@ export const supportItems = (answers: FormValue): Array<KeyValueItem> => {
     ...baseItems2,
   ]
 }
+
+export const payerItems = (answers: FormValue): Array<KeyValueItem> => {
+  const { payer, payerName, payerNationalId, payerEmail } =
+    getApplicationAnswers(answers)
+
+  const baseItems: Array<KeyValueItem> = [
+    {
+      width: 'full',
+      keyText: newPrimarySchoolMessages.differentNeeds.payerSubSectionTitle,
+      valueText:
+        payer === PayerOption.APPLICANT
+          ? newPrimarySchoolMessages.differentNeeds.payerOptionApplicant
+          : newPrimarySchoolMessages.differentNeeds.payerOptionOther,
+    },
```

Comment:
This has probably already been approved, but I think it would look cleaner if, when the payer is “Other,” we don’t show Annar under Greiðandi.

<img width="823" height="292" alt="Screenshot 2025-10-30 at 11 40 27" src="https://github.com/user-attachments/assets/dcdec206-4727-47ca-b631-2fef93aae4ba" />